tools/certs/.gitignore

```diff
@@ -0,0 +1,3 @@
+/0*.pem
```

Comment:
Add this to the root .gitignore please (some tooling we have only works with the root).